src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,127 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import classNames from 'classnames';
+import { GenderFilters, SearchQueries } from '../../types';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const toggleCentruies = (centr: string): string => {
+    const curCentr = searchParams.getAll(SearchQueries.centuries);
+    const newCentr = curCentr.includes(centr)
+      ? curCentr.filter(c => c !== centr)
+      : [...curCentr, centr];
+
+    return getSearchWith(searchParams, { [SearchQueries.centuries]: newCentr });
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        {Object.keys(GenderFilters).map(filter => {
+          const filterKey = GenderFilters[filter as keyof typeof GenderFilters];
+
+          return (
+            <Link
+              key={filterKey}
+              className={classNames({
+                'is-active': searchParams.get(SearchQueries.sex) === filterKey,
+              })}
+              to={{
+                pathname: '/people',
+                search: getSearchWith(searchParams, {
+                  [SearchQueries.sex]: filterKey,
+                }),
+              }}
+            >
+              {filter
+                .split('')
+                .map((ch, i) => (i == 0 ? ch.toLocaleUpperCase() : ch))}
```

Comment:
Code Quality Issue: Avoid using magic numbers. Here '0' is a magic number. It would be better to create a constant for it with a descriptive name, so the code is more readable and maintainable.